components/CommunityList/CommunityList.js

```diff
@@ -28,7 +28,6 @@ function CommunityList(props = {}) {
     <Box display="flex" flexWrap="wrap" justifyContent="center" margin="-8px">
       {featuredItems.map((item, i) => (
         <CustomLink
-          as="a"
```

Comment:
Another question/gut check from @drewbarontini  — I removed this `as="a"` on the `<CustomLink>` because, from my understanding, that component will wrap it's child (via `Component` prop)  in a `<Link>` and I _assume_ ultimately put a Next-friendly `<a>` somewhere in the view tree?

Reply:
Yeah, this is just an annoyance of the `next/link` component. It has a `<Link>` and then needs an `<a>` inside (with no `href`!), so if you want to attach handlers to the `<a>` itself, it's done through that `<CustomLink>` component instead. I _really_ wish there were a better solution, which maybe there is…